tests/scripts/run_profiler_regressions.sh

```diff
@@ -5,28 +5,30 @@ source scripts/tools_setup_common.sh
 set -eo pipefail
 
 run_additional_T3000_test(){
-    remove_default_log_locations
-    mkdir -p $PROFILER_ARTIFACTS_DIR
+    if [ "$ARCH_NAME" == "wormhole_b0" ]; then
```

Comment:
We don't allow such if statements anymore

Please skip the test using fixtures

Reply:
All for not allowing these ifs, are the pytest skip fixtures in?